docs/source-2.0/spec/idl.rst

```diff
@@ -193,29 +191,15 @@ string support defined in :rfc:`7405`.
     EnumTypeName            :%s"enum" / %s"intEnum"
     EnumShapeMembers        :"{" [`WS`] 1*(`TraitStatements` `Identifier` [`ValueAssignment`] [`WS`]) "}"
     ValueAssignment         :[`SP`] "=" [`SP`] `NodeValue` `BR`
-    ListStatement           :%s"list" `SP` `Identifier` [`Mixins`] [`WS`] `ListMembers`
-    ListMembers             :"{" [`WS`] [`ListMember`] [`WS`] "}"
-    ListMember              :`TraitStatements` (`ElidedListMember` / `ExplicitListMember`)
-    ElidedListMember        :%s"$member"
-    ExplicitListMember      :%s"member" [`SP`] ":" [`SP`] `ShapeId`
-    MapStatement            :%s"map" `SP` `Identifier` [`Mixins`] [`WS`] `MapMembers`
-    MapMembers              :"{" [`WS`] [`MapKey` / `MapValue` / (`MapKey` `WS` `MapValue`)] [`WS`] "}"
-    MapKey                  :`TraitStatements` (`ElidedMapKey` / `ExplicitMapKey`)
-    MapValue                :`TraitStatements` (`ElidedMapValue` / `ExplicitMapValue`)
-    ElidedMapKey            :%s"$key"
-    ExplicitMapKey          :%s"key" [`SP`] ":" [`SP`] `ShapeId`
-    ElidedMapValue          :%s"$value"
-    ExplicitMapValue        :%s"value" [`SP`] ":" [`SP`] `ShapeId`
+    AggregateShapeStatement :`AggregateShapeTypeName` `SP` `Identifier` [`Mixins`] `StructureMembers`
```

Comment:
Rather than continue to call these `StructureMembers` we should probably rename it to `Members` or similar, and then that contains a `Member` production that supports value assignment. That way we don't have, say, a list shape that contains `StructureMembers`. The associated productions like `ExplicitStructureMember` would also be updated.